### PR TITLE
[WIP] working on mixtool install

### PR DESCRIPTION
**Pull request by allenmqcymp**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
 /_output/
+<<<<<<< HEAD
 /mixtool
 .vscode
+=======
+/mixtool
+>>>>>>> bd0efc3... add mixtool list functionality (#28)
```
Comment (metalmatze) [inline .gitignore]: Seems like you committed a git merge conflict by accident. :upside_down_face:

File: cmd/mixtool/install.go [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+	mixinBaseDirectory := filepath.Join(directory)
+
+	err := os.Chdir(mixinBaseDirectory)
+	if err != nil {
+		return nil, fmt.Errorf("Cannot cd into directory %s", err)
+	}
+
+	files, err := filepath.Glob("*")
+	fmt.Println("in generatemixin, directory is ", directory, files)
```
Comment (metalmatze) [inline cmd/mixtool/install.go:98]: Is this more of a debugging statement? Do you want to get rid of it or keep it?

File: cmd/mixtool/install.go [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+	if err != nil {
+		return nil, fmt.Errorf("url parse %v", err)
+	}
+
+	// absolute directory is the same as the download url stripped of the scheme
+	absDirectory := path.Join(u.Host, u.Path)
+	absDirectory = strings.TrimLeft(absDirectory, "/:")
+	absDirectory = strings.TrimRight(absDirectory, ".git")
+
+	fmt.Println("absDirectory is", absDirectory)
```
Comment (metalmatze) [inline cmd/mixtool/install.go:113]: Same as above

**Review by metalmatze — APPROVED** [review]

**Comment by tomwilkie** [comment]
This broke CI: https://app.circleci.com/pipelines/github/monitoring-mixins/mixtool/54/workflows/527ebcb5-77e9-4174-9d88-a52f2afc6496/jobs/155

I've update circle CI to build forked projects, hopefully that'll mean CI runs on PRs like this.

**Pull request merged**